manual_correction.py

```diff
@@ -897,6 +1022,7 @@ def main():
                                 # a JSON file
                                 update_json(fname_out, name_rater, json_metadata)
                             # Generate QC report
+                            #generate_qc(fname, fname_out, task, fname_qc, subject, args.config, args.qc_lesion_plane, suffix_dict)
```

Comment:
@rohanbanerjee, I believe that this line was commented on only for debugging the fmri case. In other cases, we want to generate QC. Thus, I believe a condition checking `if contrast = 'func'` should be added here.